docs/source/Primitives.ipynb

```diff
@@ -0,0 +1,179 @@
+    "circ.cx(0, 2)\n",
+    "circ.measure_all()\n",
+    "\n",
+    "# Show circuit\n",
+    "print(circ.draw(fold=-1))\n",
+    "\n",
+    "# Initialize sampler\n",
+    "sampler = DDSIMBackendSampler()\n",
+    "\n",
+    "# Submit job\n",
+    "job = sampler.run(circ)\n",
+    "result = job.result()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "563cd527",
+   "metadata": {},
+   "source": [
+    "The `result()` method of the job returns a `SamplerResult` object, which includes both the quasi-probability distribution and job metadata."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "4820227f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(f\">>> {result}\")\n",
+    "print(f\"  > Quasi-probability distribution: {result.quasi_dists[0]}\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e74223d5",
+   "metadata": {},
+   "source": [
+    "You can also have a list with multiple quantum circuits as an input:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "03a3e980",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Create second circuit\n",
+    "circ2 = QuantumCircuit(3)\n",
+    "for qubit in circ2.qubits:\n",
+    "    circ2.h(qubit)\n",
+    "circ2.measure_all()\n",
+    "\n",
+    "circuits = [circ, circ2]\n",
+    "\n",
+    "for cir in circuits:\n",
+    "    print(cir.draw(fold=-1))\n",
+    "\n",
+    "sampler = DDSIMBackendSampler()\n",
+    "job = sampler.run(circuits)\n",
+    "result = job.result()\n",
+    "\n",
+    "print(f\">>> Quasi-probability distribution: {result.quasi_dists}\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a598052e",
+   "metadata": {},
+   "source": [
+    "Or parametrized circuits:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "55d65e6c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from qiskit.circuit import Parameter\n",
+    "\n",
+    "theta_a = Parameter(\"theta_a\")\n",
+    "theta_b = Parameter(\"theta_b\")\n",
+    "param_circ = QuantumCircuit(1)\n",
+    "param_circ.rx(theta_a, 0)\n",
+    "param_circ.rz(theta_b, 0)\n",
+    "param_circ.rx(theta_a, 0)\n",
+    "param_circ.barrier()\n",
+    "param_circ.measure_all()\n",
+    "\n",
+    "print(param_circ.draw(fold=-1))\n",
+    "\n",
+    "sampler = DDSIMBackendSampler()\n",
+    "job = sampler.run([param_circ], [[np.pi / 2, np.pi]])\n",
+    "result = job.result()\n",
+    "\n",
+    "print(f\">>> Quasi-probability distribution: {result.quasi_dists}\")"
+   ]
+  }
+ ],
+ "metadata": {
+  "kernelspec": {
+   "display_name": "My Virtual Environment",
+   "language": "python",
+   "name": "venv"
+  },
+  "language_info": {
+   "codemirror_mode": {
+    "name": "ipython",
+    "version": 3
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3"
+  }
```

Comment:
Yes, that is because I had to set the kernel in my computer to work with the virtual environment where I have everything installed. 
I guess it's a matter of changing the display name to the one that is in the other notebooks.